cli/config_test.go

```diff
@@ -15,8 +15,7 @@ var (
 #WS CONFIGURATION
 host: http://daisy.org
 port: 9999
-ws_path: ws
-ws_timeup: 10
+path: ws
```

Comment:
I kinda liked `ws_path`, rather than `path`. Especially now that we have another "path", namely `app_path`.

Reply:
We can change it back as you like.